packages/admin-ui/src/radio/Radio.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import type { Meta, Story } from '@storybook/react'
+
+import type { RadioProps } from './index'
+import { Radio, useRadioState } from './index'
+
+export default {
+  title: 'admin-ui-review/radio',
+  component: Radio,
+} as Meta
+
+export const Playground: Story<RadioProps> = (args) => {
```

Comment:
This story is not okay. Even to demonstrate, we should not have a single radio on the screen. I think you should review the radio-group component

Reply:
Good catch.